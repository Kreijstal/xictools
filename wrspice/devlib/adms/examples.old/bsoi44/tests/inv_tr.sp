Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
* Modified by Darsen Lu 03/11/2009
* Modified by Tanvir Morshed 11/21/2010

.include ./nmos4p4.mod
.include ./pmos4p4.mod
.option TEMP=27C

Vpower VD 0 1.5
Vgnd VS 0 0

Vgate   Gate   VS PULSE(0v 1.5v 100ps 50ps 50ps 200ps 500ps)

MN0 VS Gate Out VS N1 W=10u L=0.18u
MP0 VD Gate Out VS P1 W=20u L=0.18u

.tran 0.01n 600ps
.print tran v(gate) v(out)

.control
run
plot v(Gate) v(Out)
.endc

.END
