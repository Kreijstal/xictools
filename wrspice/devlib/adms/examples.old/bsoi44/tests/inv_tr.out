
Circuit: Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit: Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Date: Thu Dec 16 23:37:44  2010


Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Transient Analysis  Thu Dec 16 23:37:44  2010
--------------------------------------------------------------------------------
Index   time            v(gate)         v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	0.000000e+00	1.499923e+00	
1	1.000000e-13	0.000000e+00	1.499924e+00	
2	2.000000e-13	0.000000e+00	1.499925e+00	
3	4.000000e-13	0.000000e+00	1.499927e+00	
4	8.000000e-13	0.000000e+00	1.499928e+00	
5	1.600000e-12	0.000000e+00	1.499929e+00	
6	3.200000e-12	0.000000e+00	1.499929e+00	
7	6.400000e-12	0.000000e+00	1.499927e+00	
8	1.280000e-11	0.000000e+00	1.499925e+00	
9	2.480000e-11	0.000000e+00	1.499924e+00	
10	3.680000e-11	0.000000e+00	1.499924e+00	
11	4.880000e-11	0.000000e+00	1.499924e+00	
12	6.080000e-11	0.000000e+00	1.499924e+00	
13	7.280000e-11	0.000000e+00	1.499924e+00	
14	8.480000e-11	0.000000e+00	1.499924e+00	
15	9.680000e-11	0.000000e+00	1.499924e+00	
16	1.000000e-10	0.000000e+00	1.499924e+00	
17	1.012000e-10	3.600000e-02	1.523581e+00	
18	1.036000e-10	1.080000e-01	1.554438e+00	
19	1.084000e-10	2.520000e-01	1.572596e+00	
20	1.164194e-10	4.925820e-01	1.547492e+00	
21	1.245533e-10	7.365998e-01	1.458199e+00	
22	1.365533e-10	1.096600e+00	4.639390e-01	
23	1.460329e-10	1.380988e+00	6.200631e-02	
24	1.500000e-10	1.500000e+00	4.670244e-02	
25	1.509131e-10	1.500000e+00	3.382036e-02	
26	1.527393e-10	1.500000e+00	1.514613e-02	
27	1.563918e-10	1.500000e+00	1.980346e-03	
28	1.636967e-10	1.500000e+00	-4.354729e-04	
29	1.756967e-10	1.500000e+00	3.258380e-04	
30	1.876967e-10	1.500000e+00	-6.898827e-05	
31	1.996967e-10	1.500000e+00	1.206169e-04	
32	2.116967e-10	1.500000e+00	2.905487e-05	
33	2.236967e-10	1.500000e+00	6.910535e-05	
34	2.356967e-10	1.500000e+00	5.153684e-05	
35	2.476967e-10	1.500000e+00	5.917225e-05	
36	2.596967e-10	1.500000e+00	5.576496e-05	
37	2.716967e-10	1.500000e+00	5.719593e-05	
38	2.836967e-10	1.500000e+00	5.650628e-05	
39	2.956967e-10	1.500000e+00	5.674622e-05	
40	3.076967e-10	1.500000e+00	5.650898e-05	
41	3.196967e-10	1.500000e+00	5.662069e-05	
42	3.316967e-10	1.500000e+00	5.655332e-05	
43	3.436967e-10	1.500000e+00	5.647742e-05	
44	3.500000e-10	1.500000e+00	5.642143e-05	
45	3.512000e-10	1.464000e+00	-1.924069e-02	
46	3.536000e-10	1.392000e+00	-4.117597e-02	
47	3.584000e-10	1.248000e+00	-4.732162e-02	
48	3.655289e-10	1.034134e+00	-9.562200e-04	
49	3.722328e-10	8.330148e-01	9.159001e-02	
50	3.737328e-10	7.880148e-01	1.299929e-01	
51	3.767328e-10	6.980148e-01	2.575528e-01	
52	3.827328e-10	5.180148e-01	1.045693e+00	
53	3.840412e-10	4.787633e-01	1.146738e+00	
54	3.866580e-10	4.002604e-01	1.243383e+00	

Index   time            v(gate)         v(out)          
--------------------------------------------------------------------------------
55	3.918915e-10	2.432547e-01	1.341688e+00	
56	3.997948e-10	6.154995e-03	1.408344e+00	
57	4.000000e-10	0.000000e+00	1.409104e+00	
58	4.004103e-10	0.000000e+00	1.416845e+00	
59	4.012310e-10	0.000000e+00	1.431015e+00	
60	4.028723e-10	0.000000e+00	1.453008e+00	
61	4.061550e-10	0.000000e+00	1.478671e+00	
62	4.127203e-10	0.000000e+00	1.496958e+00	
63	4.247203e-10	0.000000e+00	1.500332e+00	
64	4.367203e-10	0.000000e+00	1.499748e+00	
65	4.487203e-10	0.000000e+00	1.499916e+00	
66	4.607203e-10	0.000000e+00	1.499853e+00	
67	4.727203e-10	0.000000e+00	1.499901e+00	
68	4.847203e-10	0.000000e+00	1.499925e+00	
69	4.967203e-10	0.000000e+00	1.499892e+00	
70	5.087203e-10	0.000000e+00	1.499872e+00	
71	5.207203e-10	0.000000e+00	1.499901e+00	
72	5.327203e-10	0.000000e+00	1.499920e+00	
73	5.447203e-10	0.000000e+00	1.499894e+00	
74	5.567203e-10	0.000000e+00	1.499879e+00	
75	5.687203e-10	0.000000e+00	1.499901e+00	
76	5.807203e-10	0.000000e+00	1.499915e+00	
77	5.927203e-10	0.000000e+00	1.499896e+00	
78	6.000000e-10	0.000000e+00	1.499882e+00	

CPU time since last call: 0.020 seconds.

Total CPU time: 0.020 seconds.

Current dynamic memory usage = 266240,
Dynamic memory limit = -9547777.

